content/en/docs/contributing/release-process.md

```diff
@@ -291,4 +294,4 @@ The process for cutting a patch release is as follows:
 2. Then, continue with the instructions in [process for releasing a minor version](#process-for-releasing-a-minor-version).
 ## Links
 
- [ArtifactHub]: https://charts.jetstack.io
+ [ChartMuseum]: https://charts.jetstack.io
```

Comment:
```suggestion
[chartmuseum]: https://charts.jetstack.io
```

Reply:
I stuck with "ChartMuseum" since that seems to be how it's rendered on their site: https://chartmuseum.com/ - but I tried to be consistent with usage.